src/fireEvent.ts

```diff
@@ -20,18 +16,20 @@ const isTextInput = (element?: ReactTestInstance) => {
     filterNodeByType(element, TextInput) ||
     filterNodeByType(element, getHostComponentNames().textInput)
   );
-};
+}
 
-const isTouchResponder = (element?: ReactTestInstance) => {
-  if (!isHostElement(element)) return false;
+function isTouchResponder(element: ReactTestInstance) {
+  if (!isHostElement(element)) {
+    return false;
+  }
 
-  return !!element?.props.onStartShouldSetResponder || isTextInput(element);
-};
+  return !!element.props.onStartShouldSetResponder || isTextInput(element);
```

Comment:
[nitpick] what do you think of using Boolean() instead of `!!` ? I find it better represents what we're doing most of the time and it's easier to read and understand

Reply:
Tbh I personally prefer `!!` because it's concise, but if you find `Boolean()` more readable we can go this way.

Reply:
both are fine for me ;) (it's just a small personal preference, also because i still find `!!` a bit hacky ^^)

Reply:
With JS you get use to some hacking techniques over time that they feel normal ;-)

Reply:
haha so true